tests/dbfs/test_api.py

```diff
@@ -34,12 +36,14 @@
 from databricks_cli.dbfs.exceptions import LocalFileExistsException
 
 TEST_DBFS_PATH = DbfsPath('dbfs:/test')
+DUMMY_TIME = time.time() * 1000
```

Comment:
I think it's better to use a fixed value here, instead of the current time. This is to make the test deterministic and consistent for each run. It may not matter that much for this particular test, though.